DSharpPlus/Net/Serialization/DiscordComponentJsonConverter.cs

```diff
@@ -46,6 +46,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var cmp = type switch
             {
                 ComponentType.ActionRow => new DiscordActionRowComponent(),
+                ComponentType.Button when (int)job["style"] is 5 => new DiscordLinkButtonComponent(),
```

Comment:
Alternatively you could do .ToObject<T> but I suppose this is smaller.

Reply:
Is there any specific reason why I should use `.ToObject`? Performance, ease of use, etc

Reply:
In this case, not particularly.